main.py

```
@@ -33,7 +33,16 @@
 from pathlib import Path
 from typing import Any
 
+
 _TEST_DATA_DIR = "/test_data"
+
+
+def _get_lmeval_version() -> str:
+    try:
+        from importlib.metadata import version
+        return version("lm_eval")
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)